### PR TITLE
Fixes the harpoon gun jamming because of inheritance

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

These guns shouldn't jam. 

## Why It's Good For The Game

Inheritance makes some things not work very good if those things do different things.

Ye.

## Changelog
:cl:
fix: Fixes harpoon guns jamming because they're a child of the mosin nagant.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
can you also fix the grammar of the brand new mosin nagant's description? it currently includes the sentence "You would rather not **to** damage it."

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -123,6 +123,10 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/boltaction/harpoon
 	fire_sound = 'sound/weapons/gun/sniper/shot.ogg'
 	can_be_sawn_off = FALSE
+	can_jam = FALSE
+	jamming_chance = 0
+	unjam_chance = 0
+	jamming_increment = 0
```
Comment (Jared-Fogle) [inline code/modules/projectiles/guns/ballistic/rifle.dm:129]: Surely can_jam = FALSE should be enough? If not, the jamming code is bad.

**Comment by necromanceranne** [comment]
> can you also fix the grammar of the brand new mosin nagant's description? it currently includes the sentence "You would rather not **to** damage it."

I'll do that later if the other pr gets closed or whatever.

File: code/modules/projectiles/guns/ballistic/rifle.dm (hunk 1 above)
Comment (necromanceranne) [reply code/modules/projectiles/guns/ballistic/rifle.dm:129]: It should but I thought I'd hit these anyway.

Comment (Jared-Fogle) [reply code/modules/projectiles/guns/ballistic/rifle.dm:129]: I'd rather you didn't, it lies about the stability of can_jam 😄

Comment (necromanceranne) [reply code/modules/projectiles/guns/ballistic/rifle.dm:129]: fair

**Review by Mothblocks — APPROVED** [review]
do the description change or whatever in another pr

**Comment by necromanceranne** [comment]
If you want that PR, it's the pipegun round 2 one.

**Pull request merged**